src/java/picard/sam/AddOrReplaceReadGroups.java

```diff
@@ -30,13 +30,33 @@
  * @author mdepristo
  */
 @CommandLineProgramProperties(
-        usage = "Replaces all read groups in the INPUT file with a single new read group and assigns " +
-                "all reads to this read group in the OUTPUT BAM",
-        usageShort = "Replaces read groups in a BAM or SAM file with a single new read group",
+        usage = AddOrReplaceReadGroups.USAGE_SUMMARY + AddOrReplaceReadGroups.USAGE_DETAILS,
+        usageShort = AddOrReplaceReadGroups.USAGE_SUMMARY,
         programGroup = SamOrBam.class
 )
 public class AddOrReplaceReadGroups extends CommandLineProgram {
-
+    static final String USAGE_SUMMARY = "Replaces read groups in a BAM file.  ";
+    static final String USAGE_DETAILS = "This tool enables the user to merge all the read group in the INPUT files into a single " +
+            "new read group.  The definition of a read group can vary depending on the sequencing platform used.  " +
+            "For example, a read group refers to an instrument lane for the Illumina platform and for SOLiD, a read group indicates a slide.  " +
```

Comment:
I think the original explanation was clearer.  Can I suggest:

"This tool enables the user to replace all read groups in the INPUT file with a single new read group and assign all reads to this read group in the OUTPUT BAM."